ReactWindows/ReactNative.Shared/UIManager/UIManagerModule.cs

```diff
@@ -34,17 +43,24 @@ public partial class UIManagerModule : ReactContextNativeModuleBase, ILifecycleE
         /// <param name="reactContext">The React context.</param>
         /// <param name="viewManagers">The view managers.</param>
         /// <param name="uiImplementation">The UI implementation.</param>
+        /// <param name="window">The ApplicationView/Framework Element.</param>
         public UIManagerModule(
             ReactContext reactContext,
             IReadOnlyList<IViewManager> viewManagers,
-            UIImplementation uiImplementation)
+            UIImplementation uiImplementation,
+#if WINDOWS_UWP
+            ApplicationView window)
+#else
+            FrameworkElement window)
+#endif
```

Comment:
Can't we just pass change the constructor signature for .NET 4.6 only?  I really don't want to have to think through all the implications of not grabbing the ApplicationView when it's used.  Basically, this would be better if you parameterized constructor on the window parameter only for .NET 4.6